pkg/network/ovn_kubernetes_test.go

```diff
@@ -4052,7 +4052,7 @@ func Test_renderOVNKubernetes(t *testing.T) {
 				client:          cnofake.NewFakeClient(),
 				featureGates:    udnFeatureGate,
 			},
-			expectNumObjs: 41,
+			expectNumObjs: 42,
```

Comment:
would it make sense to add another test without the feat gate enabled and keep expecting 41 ?...

Just a nit, probably makes no sense to test the feature gate ...

Reply:
There are two other test right above this one that run with and w/o the network-segmentation FG
https://github.com/openshift/cluster-network-operator/blob/c3896980b94595541e4c8a716579f48cc71040e3/pkg/network/ovn_kubernetes_test.go#L4021-L4045